src/mame/includes/galaga.h

```diff
@@ -13,6 +13,49 @@
 #include "screen.h"
 #include "tilemap.h"
 
+
+#define LFSR_CYCLES_PER_LINE 		256
+#define VISIBLE_LINES			224
+#define STARFIELD_X_OFFSET_GALAGA	16
+#define STARFIELD_Y_OFFSET_BOSCO	16
+#define STARFIELD_PIXEL_WIDTH		256
+
+#define LFSR_HIT_MASK		0xFA14
+#define LFSR_HIT_VALUE		0x7800
+#define LFSR_SEED		0x7FFF
+
+
+static const int speed_X_cycle_count_offset[] =
+{
+	0,  1,  2,  3, -4, -3, -2, -1 
+};
+
+static const int pre_vis_cycle_count_values[] = 
+{
+	22 * LFSR_CYCLES_PER_LINE, 
+	23 * LFSR_CYCLES_PER_LINE, 
+	22 * LFSR_CYCLES_PER_LINE, 
+	23 * LFSR_CYCLES_PER_LINE, 
+	19 * LFSR_CYCLES_PER_LINE, 
+	20 * LFSR_CYCLES_PER_LINE, 
+	20 * LFSR_CYCLES_PER_LINE, 
+	22 * LFSR_CYCLES_PER_LINE
+};
+
+static const int post_vis_cycle_count_values[] = 
+{
+	10 * LFSR_CYCLES_PER_LINE, 
+	10 * LFSR_CYCLES_PER_LINE, 
+	12 * LFSR_CYCLES_PER_LINE, 
+	12 * LFSR_CYCLES_PER_LINE, 
+	9  * LFSR_CYCLES_PER_LINE, 
+	9  * LFSR_CYCLES_PER_LINE, 
+	10 * LFSR_CYCLES_PER_LINE, 
+	9  * LFSR_CYCLES_PER_LINE
+};
```

Comment:
You can't pollute the global scope in a header like this.  Also, the static arrays will be instantiated for every compilation unit that uses this header.

Reply:
Change committed